commands.go

```diff
@@ -38,10 +32,15 @@ func (b *bot) sendMessageDedupe(m string, s *dggchat.Session) {
 
 func (b *bot) staticMessage(m dggchat.Message, s *dggchat.Session) {
 
-	for command, response := range commands {
-		if strings.HasPrefix(m.Message, command) {
+	b.commandMutex.Lock()
+	defer b.commandMutex.Unlock()
 
-			b.sendMessageDedupe(response, s)
+	for _, c := range b.commands {
+		if strings.HasPrefix(m.Message, c.Call) {
+			b.sendMessageDedupe(c.Message, s)
+			c.LastUse = time.Now().UTC()
```

Comment:
Is that for some future feature, like rate-iimiting?

Reply:
can be or to see if it can be retired after no use for a long time